src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -82,11 +91,53 @@ object WdlExpression {
     val parseTree = parser.parse_e(tokens, new WdlSyntaxErrorFormatter(terminalMap))
     new WdlExpression(parseTree.toAst)
   }
+
+  def toString(ast: AstNode, highlighter: SyntaxHighlighter = NullSyntaxHighlighter): String = {
```

Comment:
Per my TDD tendencies I pretty much always favor KISS/YAGNI since I don't want to write more tests or code than I have to.    However I'd like to be able to test syntax highlighters, and parameterizing this method by a highlighter does seem to make that easily possible (hint).  :smile:  

I wouldn't want rigid compile-time static configuration as that makes testing hard.  But if Cromwell had nice idiomatic DI (*cough* reader monad *cough*) that tests could manipulate, that would be pretty sweet.